gradle/java-library.gradle

```diff
@@ -28,5 +28,13 @@ test {
     }
 }
 
+tasks.withType(Checkstyle) {
+    reports {
+        xml.enabled false
+        html.enabled true
+        html.stylesheet resources.text.fromFile("$rootDir/config/checkstyle/checkstyle.xsl")
```

Comment:
It makes searching for errors a lot easier. Else it would be thousands of rows with 0 information